lib/mailchimpHelpers.ts

```diff
@@ -0,0 +1,111 @@
+import md5 from "md5";
+
+type mergeFields = {
+  EMAIL: string,
+  FNAME: string,
+  LNAME: string, 
+  JOB?: string
+  COMPANY?: string,
+  ISNL?: string,
+  SPONSOR?: string,
+  ISPARTIC?: string,
+  CITY?: string,
+  SCHOOL?: string,
+  GRADE?: string,
+  TECHACCESS?: string,
+  P_FNAME?: string,
+  P_LNAME?: string,
+  CONSENT?: string,
+  P_EMAIL?: string,
+  REF_OTHER?: string,
+  PARENT?: string,
+  DOB?: string,
+  PARTIC_23?: string,
+  PARENT_23?: string,
+  REFERRER?: string,
+  TIER?: string,
+  VOLUNTEER?: string,
+  SUBSCRIBER?: string,
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const checkMailChimpContact = async (mergeFields: mergeFields): Promise<any> => {
```

Comment:
Not quite sure to type the `Promise<any>` here tbh. Find the interaction between Mailchimp and TypeScript a little hard to type 😓

Reply:
Don't worry about it - this repo has larger fish to fry haha